public/data/mainMovieList.json

```
@@ -0,0 +1,111 @@
+[
+  {
+    "key": 1,
+    "movieTitle": "007 노 타임 투 다이",
+    "movieRating": "12",
+    "advanceRate": "21.4%",
+    "releaseDate": "2021.09.29",
+    "moviePoster": "https://img.megabox.co.kr/SharedImg/2021/09/03/EdklwEN8i1d8A6ndsKHwkKuEUwVkGOau_420.jpg",
+    "movieLikes": "652"
```

```suggestion
    "movieLikes": 652
```
string보다는 number 타입이 자연스럽지 않을까요?

---

맞습니다. 그게 더 나아보입니다.